src/Main.hs

```diff
@@ -113,6 +113,7 @@ main = withUtf8 $ withCP65001 $ do
   slotsCache <- readIORef cacheSlotsRef
 
   -- save corpus
+  let saveCoverageInRequestedFormats a b c d e = void $ mapM (\fmt -> saveCoverage fmt a b c d e) cfg.campaignConf.coverageFormats
```

Comment:
can we let saveCoverage to handle multiple formats instead of this helper function?